### PR TITLE
feat: modify limits commands to align with sf and sfdx

### PR DESCRIPTION
**Pull request by peternhale**
@W-11976036@
removed force as the top-level topic
add aliases back to original command
add flag aliases to maintain backward compatibility
sort sobjects by name

### CODE REVIEW

File: command-snapshot.json [hunk 1]
```diff
@@ -2,13 +2,37 @@
   {
     "command": "force:limits:api:display",
     "plugin": "@salesforce/plugin-limits",
-    "flags": ["apiversion", "json", "loglevel", "targetusername"],
```
Comment (mshanemc) [inline command-snapshot.json:5]: what happens if someone supplies currently valid flags for apiversion or loglevel ?

File: command-snapshot.json [hunk 2]
```diff
@@ -2,13 +2,37 @@
...
-    "alias": []
+    "flags": ["json", "target-org"],
+    "alias": ["force:limits:api:display", "show:api:limits"]
   },
   {
     "command": "force:limits:recordcounts:display",
     "plugin": "@salesforce/plugin-limits",
-    "flags": ["apiversion", "json", "loglevel", "sobjecttype", "targetusername"],
-    "alias": []
+    "flags": ["json", "sobject-type", "target-org"],
```
Comment (mshanemc) [inline command-snapshot.json:11]: over in `data` Matt and I decided to go with `sobject` instead of `sobject-type`.

**Review by mshanemc — CHANGES_REQUESTED** [review]
also have Matt review for sf naming conventions

File: command-snapshot.json (hunk 2 above)
Comment (peternhale) [reply command-snapshot.json:11]: I will change the sf flag to `sobject` and leave the alias as is

File: command-snapshot.json (hunk 1 above)
Comment (peternhale) [reply command-snapshot.json:5]: @mshanemc No error is thrown and command finishes w/o error. I assume the same with loglevel

Comment (peternhale) [reply command-snapshot.json:5]: sf does not support them natively and these commands cannot reasonably consume api version.

**Comment by mattgraham** [comment]
@mshanemc we talked after @peternhale and I met and you mentioned you were not a fan of `sf org list api-limits`. Do we have thoughts on a better alternative? cc @mdonnalley keeping to the <product/service> <action> <resource> model.

**Comment by peternhale** [comment]
> @mshanemc we talked after @peternhale and I met and you mentioned you were not a fan of `sf org list api-limits`. Do we have thoughts on a better alternative? cc @mdonnalley keeping to the [product/service] [action] [resource] model.

@mattgraham who is not a fan?

**Comment by mshanemc** [comment]
> > @mshanemc we talked after @peternhale and I met and you mentioned you were not a fan of `sf org list api-limits`. Do we have thoughts on a better alternative? cc @mdonnalley keeping to the [product/service] [action] [resource] model.
> 
> @mattgraham who is not a fan?

`sf org list limits` is better (API limits are just one of the many rows it displays).

My other concern is `sf org list` "covering" this up making it harder to find or access.  `sf org get limits` sounds good to me since this is just a retrieve from that api.

**Comment by mattgraham** [comment]
> > > @mshanemc we talked after @peternhale and I met and you mentioned you were not a fan of `sf org list api-limits`. Do we have thoughts on a better alternative? cc @mdonnalley keeping to the [product/service] [action] [resource] model.
> > 
> > 
> > @mattgraham who is not a fan?
> 
> `sf org list limits` is better (API limits are just one of the many rows it displays).
> 
> My other concern is `sf org list` "covering" this up making it harder to find or access. `sf org get limits` sounds good to me since this is just a retrieve from that api.

Just for continued conversation knowing this has merged and that we can update as we find the best method for the overall sf pattern: I do like the idea of `sf org list limits` showing your org limits in table form:

| Description | Limit | Period |
| --- | ----------- | --- |
| API | 890/1,000 |24 Hour |
| SOQL Queries  | 60/100 | 24 Hour |
| SOQL Records | 34,000/50,000 |  24 Hour |

then we myabe we add filters in addition to them. Possibly `sf org list limits api` or `sf org list limits:api`. Thoughts?

**Pull request merged**